### ISSUE TITLE
How to evaluate a pretrained model on nuScenes validation dataset? 

### ISSUE THREAD

**Issue by ChulanZhang** [open-mmlab/mmdetection3d#1474]
I have found the instruction for running a pretrained model on the nuScenes test dataset here:
https://mmdetection3d.readthedocs.io/en/latest/datasets/nuscenes_det.html 
But don't how to run it on the validation dataset. Can someone teach me which config should I change?

**Comment by ChulanZhang** [comment]
> The test set is set to the validation set by default. If you need to change this option, please modify the pkl filename [here](https://github.com/open-mmlab/mmdetection3d/blob/master/configs/_base_/datasets/nus-3d.py#L132).

Thanks for the help! I changed that to "nuscenes_infos_test.pkl" and the test code runs without any issue. But the progress bar still shows 6019 samples which is the size of the validation dataset. May I know the size of the nuscenes test dataset and how can I change that in the progress bar?

**Comment by ChulanZhang** [comment]
> The test set is set to the validation set by default. If you need to change this option, please modify the pkl filename [here](https://github.com/open-mmlab/mmdetection3d/blob/master/configs/_base_/datasets/nus-3d.py#L132).

Sorry for the confusion. It turns out that I modified [here](https://github.com/open-mmlab/mmdetection3d/blob/76e351a7a6b0b83d7dc729bdfee5f17743c72f76/tools/test.py#L203) and did not use the test dataset. After fixing, the progress bar shows 6008 now. Is that the correct size of the nuScenes test dataset?

**Comment by Tai-Wang** [comment]
Yes, hope everything goes well after that.

**Comment by Tai-Wang** [comment]
Close due to inactivity.
